packages/gestalt/src/TableOfContents/TableOfContentsAnchor.css

```diff
@@ -0,0 +1,16 @@
+.item {
+  border-radius: 2px 8px 8px 2px;
+  padding-right: 16px;
+}
+
+.itemPadding {
+  padding-left: 12px;
+  padding-top: 13.5px;
+  padding-bottom: 13.5px;
+}
+
+.itemNestedPadding {
+  padding-left: 32px;
+  padding-top: 14.5px;
```

Comment:
Some feedback I recieved in the past was to avoid using .5 px, could we do either 14 or 15 here?

Reply:
Yeah, right. We will have to ask @hectoid. These numbers were specified in the Figma design.